pkgs/development/tools/hobbes/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, pkgs }:
+
+pkgs.stdenv.mkDerivation {
+  name = "hobbes";
+  version = "latest-432d6a2";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "morgan-stanley";
+    repo = "hobbes";
+    rev = "432d6a2e52fa023bf86345ca165b67ab44f2963e";
+    sha256 = "02jldsa6lb8vdy9axzpkldifq52w1ryw4kjfywnmp4s2hchiixlm";
+  };
+
+  nativeBuildInputs = with pkgs; [
+    cmake
+    gcc6
+    llvm_6
+    ncurses
+    python27
+    readline
+    zlib
+  ];
+
+  doCheck = true;
```

Comment:
Didn't the builder ran `make check` or whatever without this?

Reply:
Good point. It probably didn't run because the checkTarget was incorrect.

Reply:
Builder didn't run the tests without this flag (even with the right target).